### DISCUSSION TITLE
Exposure to algorithmic issues

### DISCUSSION THREAD

**Discussion by QZheng82** [Anura3D/Anura3D_OpenSource#86] [Q&A] [answered]
I am trying to incorporate anti-slip piles into a rock slide, but during operation, material points in the rock will pass through the defined anti-slip pile material. Is it possible that I have not turned on the contact algorithm?
![E{`O@KXZ6UACD7`7N)FZB7A](https://user-images.githubusercontent.com/119284513/226533350-b73d8afa-50f4-4e1f-b448-bf50e826fe11.png)
![O~$9~FAW70IY%FHS_PFXIPP](https://user-images.githubusercontent.com/119284513/226533359-f140b21b-c050-4e55-8b5b-9dcf1e57b1de.png)
![PPG5DOJJ9`$V7UDM1@5ZXRU](https://user-images.githubusercontent.com/119284513/226533375-3bb9ec01-e09b-4f97-918f-c9237ce58e3f.png)

**Comment by FrancescaCeccato** [comment, marked as answer]
Hi, I can't see clearly your figure but I have the feeling that the problem is not collectly discretized. It seems that the pile is overlapped with the slope surface. Try to remake the geometry making sure that surfaces are not overlapping.

**Reply by QZheng82** [reply]
Thank you! The problem is solved.
